TranslatorReasonerAPI.yaml

```diff
@@ -820,6 +820,5 @@ components:
       required:
         - name
         - id
-        - operator
```

Comment:
Why remove operator requirement from constraint? I can't think of a valid constraint that does not contain an operator. 

If the driving factor was that operator had a default value, per #239, consider removing default value from operator? A constraint definition should be unambiguous anyway.